### ISSUE TITLE
UMD not detected by `no-common`

### ISSUE THREAD

**Issue by benmosher** [benmosher/eslint-plugin-import#9]
Specifically, Immutable.js's UMD preamble does not trigger a `no-common` report:
```javascript
function (global, factory) {
  typeof exports === 'object' && typeof module !== 'undefined' ? module.exports = factory() :
  typeof define === 'function' && define.amd ? define(factory) :
  global.Immutable = factory()
}...
```
